contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,69 @@
+</ComparisonTable>
+
+## When should you use web analytics?
+
+Web analytics gives you a single, pre-built [dashboard](https://us.posthog.com/web) for a high-level overview of your site. It's about collecting data in the aggregate, as opposed to tracking specific interactions from specific users.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_dark_mode_2024_10_6aa6dc9aeb.png"
+    alt="Web analytics dashboard"
+    classes="rounded"
+/>
+<Caption>Web analytics dashboard</Caption>
+
+You can use web analytics to find out (click the 👀 emojis to see visual examples):
+
+- how many [visitors](/docs/web-analytics/dashboard) there were in the last _X_ days, weeks, or months <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- which [pages](/docs/web-analytics/dashboard#paths) people are visiting <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- the [bounce rate](/docs/web-analytics/dashboard#bounce-rate) <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- all the different [sources of traffic](/docs/web-analytics/dashboard#referrers-channels-utms) to your website <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- where visitors are [located geographically](/docs/web-analytics/dashboard#world-map) <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- which device types, browsers, and operating systems visitors are using <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- how long sessions last on average <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
+- how many people converting <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
```

Comment:
I assume these images are placeholders (they are all the same)
```suggestion
- How many [visitors](/docs/web-analytics/dashboard) there were in the last _X_ days, weeks, or months <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- Which [pages](/docs/web-analytics/dashboard#paths) people are visiting <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- The [bounce rate](/docs/web-analytics/dashboard#bounce-rate) <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- All the different [sources of traffic](/docs/web-analytics/dashboard#referrers-channels-utms) to your website <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- Where visitors are [located geographically](/docs/web-analytics/dashboard#world-map) <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- Which device types, browsers, and operating systems visitors are using <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- How long sessions last on average <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
- How many people converting <details style="display:inline;"><summary style="padding:0;">👀</summary>![Screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png)</details>
```

Reply:
Yes you're right they were just placeholders to get the layout right. I can capture some screenshots and/or gifs and add them in here.